README.md

```diff
@@ -100,11 +100,11 @@ We have created example applications to document some of the ways you could incl
 
 ## Development
 
-We’ve created a simple, hot-reloading playground for development on these components. You can edit the `playground/Playground.tsx` file to import the components you are working on, and run `yarn dev` in order to start the development server. Please do not commit your work on the playground so that it is pristine for other developers to work on.
+We’ve use Storybook to create a simple, hot-reloading playground for development on these components. You can edit the `playground/Playground.tsx` file to import the components you are working on, and run `yarn dev` in order to start the development server. Please do not commit your work on the playground so that it is pristine for other developers to work on.
 
 ### Testing on mobile or a virtual machine
 
-1.  Run `yarn dev:host`
+1.  Run `yarn dev`
```

Comment:
storybook binds on all IP addresses so we don't need a special command to run on a particular IP anymore